source/WB-CAN-Firmware.c

```diff
@@ -196,10 +293,34 @@ static void testTask(void *pv) {
 	}
 }
 
+
+// * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * //
+// * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * //
+
+
+
+/*******************************************************************************
+ * Tasks
+ ******************************************************************************/
+static void blinkTask(void *pv) {
+	while(1) {
+
+		vTaskDelay(pdMS_TO_TICKS(500));
+//		printf("BLINK");
+	}
+}
+
+
 static void actuatorTask(void *pv){
 	uint32_t period = 100;
 	for(;;){
-		PWM(period, duty_cycle); // Uses Global Variable changed in Control Task
+		if (uxSemaphoreGetCount( semaphore_PWMActive ) == 1) {
+			PWM(period, duty_cycle); // Uses Global Variable changed in Control Task
+		}
+		else if(uxSemaphoreGetCount( semaphore_PWMActive ) == 0){
```

Comment:
if it's binary, there's only one other option, so no need to perform the check. Although if you want to the `== 0` is redundant as above.
```suggestion
		else {
```